datahub-web-react/src/app/onboarding/onboarding.tsx

```diff
@@ -0,0 +1,693 @@
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/glossary/business-glossary"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ENTITY_PROFILE_DOMAINS_ID,
+        selector: `#${ENTITY_PROFILE_DOMAINS_ID}`,
+        title: 'Domain',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    You can view and set this asset&apos;s <strong>Domain</strong> here.
+                </p>
+                <p>
+                    <strong>Domains</strong> are collections of related data assets associated with a specific part of
+                    your organization, such as the <strong>Marketing</strong> department.
+                </p>
+                <p>
+                    Learn more about <strong>Domains</strong>{' '}
+                    <a target="_blank" rel="noreferrer noopener" href="https://datahubproject.io/docs/domains">
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ADMIN_CREATE_INGESTION_SOURCE_ID,
+        selector: `#${ADMIN_CREATE_INGESTION_SOURCE_ID}`,
+        title: 'Create a new Ingestion Source',
+        content: (
+            <Typography.Paragraph>
+                <p>
+                    Click here to configure new Integrations from DataHub to your <strong>Data Platforms</strong>,
+                    including Transactional Databases like <strong>MySQL</strong>, Data Warehouses like{' '}
+                    <strong>Snowflake</strong>, Dashboarding tools like <strong>Looker</strong>, and many more!
+                </p>
+                <p>
+                    Learn more about ingestion and view the full list of supported Integrations{' '}
+                    <a
+                        target="_blank"
+                        rel="noreferrer noopener"
+                        href="https://datahubproject.io/docs/metadata-ingestion"
+                    >
+                        {' '}
+                        here.{' '}
+                    </a>
+                </p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: ADMIN_REFRESH_INGESTION_SOURCES_ID,
+        selector: `#${ADMIN_REFRESH_INGESTION_SOURCES_ID}`,
+        title: 'Refresh Ingestion Pipelines',
+        content: (
+            <Typography.Paragraph>
+                <p>Click here to refresh and check whether new ingestion pipelines have been set up.</p>
+                <p>You can view both pipelines created on this page and those set up using the DataHub CLI.</p>
+            </Typography.Paragraph>
+        ),
+    },
+    {
+        id: BUSINESS_GLOSSARY_INTRO_ID,
+        title: 'Business Glossary 📖',
```

Comment:
I really like this. See how you don't put "Page" at the end? Let's do that everywhere.. I also like the Emoji